<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AppletView</class>
 <widget class="QWidget" name="AppletView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>502</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel8" >
       <property name="text" >
        <string>&amp;Search:</string>
       </property>
       <property name="buddy" >
        <cstring>appletSearch</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="appletSearch" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>&lt;qt>Type here some text to filter on the applet names and comments&lt;/qt></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>S&amp;how:</string>
       </property>
       <property name="buddy" >
        <cstring>appletFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="appletFilter" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis" >
        <string>&lt;qt>Select here the only applet category that you want to show&lt;/qt></string>
       </property>
       <item>
        <property name="text" >
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Applets</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Special Buttons</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListView" name="appletListView" >
       <property name="autoScroll" >
        <bool>false</bool>
       </property>
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="movement" >
        <enum>QListView::Static</enum>
       </property>
       <property name="isWrapping" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Fixed</enum>
       </property>
       <property name="viewMode" >
        <enum>QListView::ListMode</enum>
       </property>
       <property name="uniformItemSizes" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>284</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>appletSearch</tabstop>
  <tabstop>appletFilter</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
